lib/portfinder.js

```diff
@@ -27,9 +27,9 @@ internals.testPort = function(options, callback) {
     options = {};
   }
 
-  options.port   = options.port   || exports.basePort;
-  options.host   = options.host   || null;
-  options.server = options.server || net.createServer(function () {
+  options.port   = +options.port   || exports.basePort;
```

Comment:
wow - yeah this is really a good find - odd thing is that your the first to catch it. most libs must coerce their cli args to a type before passing the port as an option.

do u mind using `Number(options.port) || Number(exports.basePort)` - aka, wrap both - u can set the default, surely someone will set it as a string someday - and the `+` coercion is less obvious when reading the code...

will merge immediately and release after, thanks again